routers/web/repo/repo.go

```diff
@@ -352,6 +352,9 @@ func Action(ctx *context.Context) {
 		ctx.Data["IsStaringRepo"] = repo_model.IsStaring(ctx, ctx.Doer.ID, ctx.Repo.Repository.ID)
 	}
 
+	// see the `hx-trigger="refreshUserCards ..."` comments in tmpl
+	ctx.RespHeader().Add("HX-Trigger", "refreshUserCards")
```

Comment:
```suggestion
	ctx.RespHeader().Add("hx-trigger", "refreshUserCards")
```

There is no point in trying to uppercase http headers because they are case-insensitive, and with this, is greppable case-sensitively.